File: Heat/ViewModels/ManifacturerModels/CreateManifacturerModelViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.ManifacturerModels

    Public Class CreateManifacturerModelViewModel
        <Required> _
        <Display(name:="Marca")> _
        Property ManifacturerID As Integer
        Property ManifacturerList As IEnumerable(Of SelectListItem)
        <Required> _
        Property Model As String
    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/ManifacturerModels/CreateManifacturerModelViewModel.cs
using System.Collections.Generic;
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.ManifacturerModels
{

    public class CreateManifacturerModelViewModel
	{
		[Required()]
		[Display(Name = "Marca")]
		public int ManifacturerID { get; set; }
		public IEnumerable<SelectListItem> ManifacturerList { get; set; }
		[Required()]
		public string Model { get; set; }
	}

}
